src/Scope/ScopeEnum.php

```diff
@@ -39,6 +39,11 @@ class ScopeEnum extends Enum
         self::PHONING_NATIONAL_MANAGER,
     ];
 
+    public const LOCAL_SCOPES = [
```

Comment:
The name is confusing, because there are another local scopes not present in the constant.
What about `SURVEY_SCOPES`? It corresponds better to its use.

Reply:
It is not just for the surveys that I added this constant. But yes, i can rename it to LOCAL_SURVEY_SCOPES if you want, but I think we'll need it very soon

Reply:
Not sure. We have `NATIONAL_SCOPES` and at the moment the scope that is not national is local. And any check for local/national scope can be done with.